src/commons/api/admin.rs

```diff
@@ -565,28 +565,45 @@ impl AddChildRequest {
 pub struct UpdateChildRequest {
     id_cert: Option<IdCert>,
     resources: Option<ResourceSet>,
+
+    #[serde(skip_serializing_if = "Option::is_none")]
+    suspend: Option<bool>,
 }
 
 impl UpdateChildRequest {
-    pub fn new(id_cert: Option<IdCert>, resources: Option<ResourceSet>) -> Self {
-        UpdateChildRequest { id_cert, resources }
+    pub fn new(id_cert: Option<IdCert>, resources: Option<ResourceSet>, suspend: Option<bool>) -> Self {
+        UpdateChildRequest {
+            id_cert,
+            resources,
+            suspend,
+        }
     }
     pub fn id_cert(id_cert: IdCert) -> Self {
         UpdateChildRequest {
             id_cert: Some(id_cert),
             resources: None,
+            suspend: None,
         }
     }
 
     pub fn resources(resources: ResourceSet) -> Self {
         UpdateChildRequest {
             id_cert: None,
             resources: Some(resources),
+            suspend: None,
+        }
+    }
+
+    pub fn suspend(suspend: bool) -> Self {
```

Comment:
Elsewhere you have unsuspend, would it be more consistent to add another method here called unsuspend rather than take a boolean flag?